src/vuestic-theme/vuestic-sass/overrides/_custom-libs.scss

```diff
@@ -112,10 +112,10 @@
   }
 }
 
-//v-tooltip
+//v-tooltip - check this one please, looks like it does nothing, though not entirely sure
 
 .tooltip.va-tooltip {
-  @extend .tooltip.show;
+  @extend .tooltip.show !optional;
```

Comment:
Ok. So marking these as optional definitely doesn't help. Everything will be broken, just silently.